pages/index.tsx

```diff
@@ -86,14 +119,38 @@ export default function Home() {
     setIsAnyKindOfErrorMessage(errorMessage);
   }
 
+  async function handleLogout() {
+    const request = await axios.post('/api/auth/logout');
+    if (request) await router.push('/login').catch((e) => console.error(e));
+  }
+
   return (
     <main className="bg-white w-full max-w-full h-screen">
+      <div className="justify-center flex">
+        <div className="bg-gray-300 sm:pr-0 pr-2 p-3 flex flex-row justify-between sm:w-72 w-full relative">
+          <div className="text-lg select-none">Lahjaidealista</div>
+          <SvgUser
+            width={32}
+            height={32}
+            className={`cursor-pointer hover:stroke-yellow-600 ${showUserWindow ? 'stroke-yellow-600' : ''} z-[98]`}
```

Comment:
Tämän kohdan `${showUserWindow ? 'stroke-yellow-600' : ''}` idea tässä on korjata `hover`in ongelma, kun käyttää sivua puhelimella. Virhe tapahtui siinä, kun `user`-iconia painoi jäi se värilliseksi vaikka ei ollut aktiivinen. Oli hiukan hämäävä. 

`z-[98]` tarkoitus on taasen tuoda `user`-icon esille, jotta sitä pystytään klikkaamaan vaikka "closeModal-wrapperi" on päällä. Siitä alempana lisää

Reply:
Korjaako tuo ongelman? Mobiilissa hover tosiaan jää päälle, mutta eikö tuossa määritetä sama keltainen uudestaan? 🤔 

Jos ymmärsin ongelman oikein, niin sen saa korjattua lisäämällä seuraavan koodin Tailwindin config-tiedostoon:

```js
  future: {
    hoverOnlyWhenSupported: true,
  },
```

Siitä tulee Tailwindin v4:ssä oletusasetus, mutta v3:ssa se pitää laittaa käsin päälle. Lisätietoa [Tailwindin repon PR:stä](https://github.com/tailwindlabs/tailwindcss/pull/8394).

Reply:
> Tarvitaanko ${showUserWindow ? 'stroke-yellow-600' : ''} sitten enää?

`${showUserWindow ? 'stroke-yellow-600' : ''}` tämä kohta tekee sen, että käyttäjäikonin väri pysyy keltaisena kun modal on auki. Ja kun modal suljetaan, muuttuu käyttäjäikoni takaisin mustaksi. En tiedä tekeekö tuo siitä yhtään järkevämmän näköistä kylläkään :sweat_smile: